src/vuestic-theme/vuestic-components/vuestic-modal/VuesticModal.demo.vue

```diff
@@ -0,0 +1,112 @@
+      <vuestic-radio-button value="large" v-model="modalSize" option="large"/>
+    </div>
+    <vuestic-modal
+      v-model="isModalShow"
+      title="Simple Popup, Full Width"
+      message="About 64% of all on-line teens say that do things online that they wouldn’t want their parents to know about.   11% of all adult internet users visit dating websites and spend their time Some of the classify their behavior as “cyber affair”   More then 60% of employees use company PC for the personal needs during their work hours as long as 80 minutes per day."
+      okText="Accept"
+      cancelText="Decline"
+      :position="modalPosition"
+      :size="modalSize"
+    />
+  </div>
+  <div class="demo-container__item">
+    <p>fullscreen modal with, slots use</p>
+    <button @click="triggerFullscreenModal">Show modal</button>
+    <vuestic-modal
+      v-model="isFullScreenModalShow"
+      :fullscreen="true">
+      <template slot="header">
+        <h2>Step 2. Centered Layout</h2>
+      </template>
+      <slot>
+        <div>About 64% of all on-line teens say that do things online that they wouldn’t want their parents to know about.   11% of all adult internet users visit dating websites and spend their time Some of the classify their behavior as “cyber affair”   More then 60% of employees use company PC for the personal needs during their work hours as long as 80 minutes per day.</div>
+      </slot>
+    </vuestic-modal>
+  </div>
+  <div class="demo-container__item">
+    <p>no-outside-dismiss, no-esc-dismiss, max-width: 800px, max-height: 250px, with scroll</p>
+    <button @click="triggerActionsModal">Show modal</button>
+    <vuestic-modal
+      v-model="isActionsModalShow"
+      :no-outside-dismiss="true"
+      :no-esc-dismiss="true"
+      max-width="800px"
+      max-height="250px"
+      title="Step 2. Centered Layout"
+      message="About 64% of all on-line teens say that do things online that they wouldn’t want their parents to know about.   11% of all adult internet users visit dating websites and spend their time Some of the classify their behavior as “cyber affair”   More then 60% of employees use company PC for the personal needs during their work hours as long as 80 minutes per day. About 64% of all on-line teens say that do things online that they wouldn’t want their parents to know about.   11% of all adult internet users visit dating websites and spend their time Some of the classify their behavior as “cyber affair”   More then 60% of employees use company PC for the personal needs during their work hours as long as 80 minutes per day."
+    />
+  </div>
+  <div class="demo-container__item">
+    <p>fixed layout && onOk, onCancel functions</p>
+    <button @click="triggerFixedLayoutModal">Show modal</button>
+    <vuestic-modal
+      v-model="isFixedLayoutModalShow"
+      max-width="800px"
+      max-height="250px"
+      :fixedLayout="true"
+      v-on:ok="onOk"
```

Comment:
You can use shortcut: `v-on:ok="onOk"` vs `@ok="onOk"`. Saves a couple of extra characters 😄